src/loaders/stm_load.c

```diff
@@ -20,6 +20,11 @@
  * THE SOFTWARE.
  */
 
+#ifdef __native_client__
+#include <sys/syslimits.h>
```

Comment:
Do we support native client? Wasn't it deprecated in Chrome some time ago in favor of wasm?

Reply:
I think that is correct, it is deprecated.

Reply:
I forgot to remove these lines from previous 3 years old patch.